lib/state/appActions.js

```diff
@@ -257,6 +258,13 @@ appActions.login = function(credentials, options, cb) {
       return cb(err);
     }
     self._logMetric(self.trackedState.LOGIN_SUCCESS);
+    
+    var acks = localStore.getItem('acknowledgments');
```

Comment:
As for explaining why. I am using the `initialState.acknowledgments` to be the defaultValue if `localStorage.acknowledgments` is not set. Because it is a nested object, with possibly more objects inside it I didn't want to redefine the initial empty structure of this object, so I am re-using it.